addins/ShortcutSample/ShortcutSample.xml

```diff
@@ -0,0 +1,154 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
I don't see a `README.md` in this PR. It is important. Look at the READMEs we have here, and pick one that fits your purpose.